src/regraderprocess.cpp

```diff
@@ -74,133 +74,6 @@ RegraderProcess::~RegraderProcess() {
     delete limiter;
 }
 
-/* public methods */
-
-void RegraderProcess::process( float** inBuffer, float** outBuffer, int numInChannels, int numOutChannels,
```

Comment:
Moved to a template as inBuffer and outBuffer can be either one of _float_ or _double_.